pom.xml

```diff
@@ -24,17 +23,20 @@
         <commons-io-version>2.7</commons-io-version>
         <commons-lang3-version>3.8.1</commons-lang3-version>
         <rx-version>2.2.7</rx-version>
-        <mockito-version>3.11.2</mockito-version>
+        <mockito-version>3.12.4</mockito-version>
         <powermock-version>2.0.9</powermock-version>
         <maven-deploy-plugin-version>3.0.0-M1</maven-deploy-plugin-version>
         <nexus-staging-plugin-version>1.6.8</nexus-staging-plugin-version>
         <maven-gpg-plugin-version>3.0.1</maven-gpg-plugin-version>
 
         <!-- versions of transitive dependencies we need to override to avoid vulnerability alerts -->
         <junit-version>4.13.2</junit-version>
+
+        <!-- This property should be empty by default. -->
+        <surefireJvmArgs></surefireJvmArgs>
```

Comment:
By default, we will use no additional JVM args when running tests via the surefire plugin.